biomage/rds/migrate.py

```diff
@@ -0,0 +1,92 @@
+import os
+import subprocess
+
+import click
+
+from ..utils.AuroraClient import AuroraClient
+from ..utils.constants import DEFAULT_AWS_ACCOUNT_ID, DEVELOPMENT, STAGING
+
+# Assuming that biomage-utils and iac root folders are located in the same folder
+MODULE_PATH = os.path.dirname(os.path.abspath(__file__))
+DEFAULT_IAC_PATH = os.path.join(MODULE_PATH, "../../../iac")
+IAC_PATH = os.getenv("BIOMAGE_IAC_PATH", DEFAULT_IAC_PATH)
+
+
+def _migrate(command, iac_path, migration_env):
+    knex_command = ["node_modules/.bin/knex", "--cwd", iac_path] + list(command)
+
+    proc = subprocess.Popen(
+        knex_command,
+        cwd=iac_path,
+        env=migration_env,
+    )
+    proc.wait()
+
+
+@click.command()
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=DEVELOPMENT,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+@click.option(
+    "-s",
+    "--sandbox_id",
+    required=False,
+    default=None,
+    show_default=True,
+    help="Sandbox id to migrate to. Required if migrating to staging",
+)
+@click.option(
+    "--iac_path",
+    required=False,
+    default=IAC_PATH,
+    show_default=True,
+    help="Path to the IAC folder",
+)
+@click.argument("command", required=False, nargs=-1)
+def migrate(iac_path, sandbox_id, input_env, command):
+    """
+    Runs knex migration command (default to migrate:latest) in local or staged env.
+
+    Examples.:\n
+        biomage rds migrate -i development\n
```

Comment:
considering `-i development` is the default value, I would remove that part from the example.
I would also add another one which will probably be the most used after that
`biomage rds migrate migrate:down`